### DISCUSSION TITLE
Lsposed1.8.3 build failed

### DISCUSSION THREAD

**Discussion by xyxdaily** [LSPosed/LSPosed#2053] [Q&A]
![image](https://user-images.githubusercontent.com/50726457/181045859-d44a5035-cfac-4b93-9971-49eb5a72da56.png)
could you help me.
My Java: sudo apt install openjdk-17-jre-headless

**Comment by yujincheng08** [comment]
`git submodule update --init --recursive`

**Reply by xyxdaily** [reply]
I have done this already, I will try again.

**Comment by xyxdaily** [comment]
I meet another strange error.
`gradlew zipZygiskDebug` is ok,
however, `gradlew zipZygiskRelease` failed.
![image](https://user-images.githubusercontent.com/50726457/181431789-6dd2b2bc-2211-4bd7-b5a1-74e0c57c2878.png)
